backend/api/professor/grading_materials.py

```
@@ -0,0 +1,333 @@
+        return {
+            "message": "Grading material queued for reprocessing",
+            "material_id": material_id
+        }
+        
+    except HTTPException:
+        raise
+    except Exception as e:
+        debug_log(f"[GRADING_MATERIALS] Reprocess error: {str(e)}")
+        raise HTTPException(status_code=500, detail="Failed to reprocess grading material")
+
+async def _process_grading_material_async(material_id: int, content: str, filename: str):
```

Ruff (0.14.1)

Missing return type annotation for private function `_process_grading_material_async`

Add return type annotation: `None`

(ANN202)